internal/services/sentinel/sentinel_alert_rule_machine_learning_behavior_analytics_test.go

```diff
@@ -191,17 +203,9 @@ resource "azurerm_log_analytics_workspace" "test" {
   sku                 = "PerGB2018"
 }
 
-resource "azurerm_log_analytics_solution" "test" {
-  solution_name         = "SecurityInsights"
-  location              = azurerm_resource_group.test.location
-  resource_group_name   = azurerm_resource_group.test.name
-  workspace_resource_id = azurerm_log_analytics_workspace.test.id
-  workspace_name        = azurerm_log_analytics_workspace.test.name
-
-  plan {
-    publisher = "Microsoft"
-    product   = "OMSGallery/SecurityInsights"
-  }
+resource "azurerm_sentinel_log_analytics_workspace_onboarding" "test" {
+  resource_group_name = azurerm_resource_group.test.name
+  workspace_name      = azurerm_log_analytics_workspace.test.name
```

Comment:
and we can (should?) update this to simple take an id
```suggestion
resource "azurerm_sentinel_log_analytics_workspace_onboarding" "test" {
  workspace_id      = azurerm_log_analytics_workspace.test.id
```